sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -405,19 +382,19 @@ export class EventHubClient {
     if (!connectionString || (connectionString && typeof connectionString !== "string")) {
       throw new Error("'connectionString' is a required parameter and must be of type: 'string'.");
     }
-    const config = EventHubConnectionConfig.create(connectionString, path);
-
-    config.webSocket = options && options.webSocket;
-    config.webSocketEndpointPath = "$servicebus/websocket";
-    config.webSocketConstructorOptions = options && options.webSocketConstructorOptions;
-
+    const config = ConnectionConfig.create(connectionString, path);
     if (!config.entityPath) {
-      throw new Error(
-        `Either the connectionString must have "EntityPath=<path-to-entity>" or ` +
-          `you must provide "path", while creating the client`
+      throw new TypeError(
+        `Either provide "path" or the "connectionString": "${connectionString}", ` +
+          `must contain EntityPath="<path-to-the-entity>".`
```

Comment:
The previous `if` check here would never succeed as `EventHubConnectionConfig.create` was throwing an error if `entityPath` doesn't exist.

This change is to keep the existing tests passing.